discord/ext/commands/view.py

```diff
@@ -48,12 +48,77 @@
 }
 _all_quotes = set(_quotes.keys()) | set(_quotes.values())
 
+class Separator:
+    """An argument qualifier, which acts as the delimiter for arguments.
```

Comment:
This documentation should probably show some output to show the difference between `strip_ws=False` and `strip_ws=True`.